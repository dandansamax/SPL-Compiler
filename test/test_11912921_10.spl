int sum(int x, int y)
{
    return x + y;
}

int test_10()
{
    int x = 3, y = 2;
    x = sum(x, x, y);
    return comp(x, y);
}

int comp(int x, int y)
{
    return !(x < y);
}
