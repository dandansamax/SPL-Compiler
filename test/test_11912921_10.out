Error type 1 at Line 10: a variable is used without a definition: a
Error type 2 at Line 11: a function is invoked without a definition: comp
